README.md

```diff
@@ -1,2 +1,65 @@
-# ufs-coastal-app
-UFS Coastal Application
+# Introduction 
+The ufs-coastal-app repository is the umbrella for the UFS Coastal project currently under development by NOAA and NCAR, which supports coastal forecast requirements. The UFS Coastal Application is meant to constitute a workflow for coastal forecasting, wherein the coupling infrastructure is provided by [this fork of  ufs-weather-model](https://github.com/oceanmodeling/ufs-weather-model), which is currently in active development.
+
+The [model-level repo](https://github.com/oceanmodeling/ufs-weather-model) contains the model code and external links needed to build the Unified Forecast System (UFS) coastal model executable and associated model components, including the ROMS, FVCOM, ADCIRC and SCHISM plus WaveWatch III model components. 
+
+Here, model component means a forecast component (e.g. coastal ocean model) that represents a physical domain (e.g. Atlantic Ocean) or process (e.g. biogeochemistry). An application is a workflow that is designed for a particular forecasting purpose. A workflow consists of a set of model components and coupling infrastructure, provided by UFS Coastal Model, pre- and post-processing scripts, and is associated with a range of valid coupling configurations. 
+
+# Where to find more information 
+[Documentation](https://oceanmodeling.github.io/ufs-coastal-app/versions/main/html/index.html)  (in development)
+[UFS Coastal Wiki](https://github.com/oceanmodeling/ufs-coastal-model/wiki/) 
```

Comment:
I think we need to point https://github.com/oceanmodeling/ufs-weather-model/wiki now. If you don't mind could you fix it.

Reply:
fixed now